packages/core/src/components/menu/_common.scss

```diff
@@ -14,9 +14,19 @@ $menu-item-padding-large: ($pt-button-height-large - $pt-icon-size-large) / 2 !d
 $menu-background-color: $white !default;
 $dark-menu-background-color: $dark-gray4 !default;
 
+$menu-item-color-hover: $minimal-button-background-color-hover;
+$menu-item-color-active: $minimal-button-background-color-active;
+$dark-menu-item-color-hover: $dark-minimal-button-background-color-hover;
+$dark-menu-item-color-active: $dark-minimal-button-background-color-active;
+
 // customize modifier classes with params.
 // setting modifier to "" will generally apply it as default styles due to & selectors
-@mixin menu-item($disabled-selector: ".pt-disabled", $hover-selector: ":hover") {
+@mixin menu-item(
+  $active-selector: ".pt-active",
+  $disabled-selector: ".pt-disabled",
+  $active-pseudoselector: ":active",
+  $hover-pseudoselector: ":hover"
```

Comment:
variables for pseudoselectors feels wrong--there just aren't any other values than these ones. also what happens if someone doesn't want to support pseudoclasses when using this mixin? not currently possible, it will error or generate bad CSS.

another approach could be `$active-selector: "&:active, &.pt-active"`.